services/app-web/src/constants/errors.ts

```diff
@@ -17,8 +18,17 @@ const ERROR_MESSAGES = {
     question_missing_field: 'Your question is missing information.',
     question_error_generic: 'Error attempting to add question.',
     response_error_generic: 'Error attempting to add response.',
+    email_error_generic: 'Error attempting to send email.',
 }
 
 const MAIL_TO_SUPPORT = 'mc-review@cms.hhs.gov'
 
-export { MAIL_TO_SUPPORT, ERROR_MESSAGES }
+const GRAPHQL_ERROR_MESSAGES = {
```

Comment:
🔍 This mapping of graphql error codes to message is not resource agnostic.

 BAD_USER_INPUT or FORBIDDEN could come down on graphql endpoints that are related to all kinds of things (fetching users, editing users, fetching settings, fetching questions) and the messages wouldn't make sense. I suggest still relying on ERROR_MESSAGES with specifically named keys for error messages for now. Something like mapping generic errors for each graphql code seems like a change that should be part of the spike on unifying error messaging.

Reply:
That is a good point. `BAD_USER_INPUT` and `FORBIDDEN` should be more generic. 

Currently `GRAPHQL_ERROR_MESSAGES` is only used in unit tests. I guess it should be living where the tests are, but once we figure out the approach we could make it a less test-focused resource.